sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcQuerySuite.scala

```diff
@@ -844,6 +844,21 @@ abstract class OrcQuerySuite extends OrcQueryTest with SharedSparkSession {
       df.write.format("orc").save(path)
     }
   }
+
+  test("SPARK-39387: Make vectorized orc columar writer batch size configurable") {
```

Comment:
ID looks wrong to me. Could you double-check? I expected SPARK-39381.

Reply:
Thanks, i fixed it now.